### PR TITLE
2450 positions lazy load

### PR DESCRIPTION
**Pull request by stephensprinkle**

### CODE REVIEW

File: src/modules/common/components/spinner.jsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Spinner = () => (
+  <article className="spinner-container" >
+    <div className="spinner" />
+  </article>
+);
```
Comment (tinybike) [inline src/modules/common/components/spinner.jsx]: Lovely :grinning:

**Comment by tinybike** [comment]
I'm having some difficulty with the close position button -- I click it, the spinner starts spinning, but the position is never actually closed (I don't see any transactions firing).

**Comment by tinybike** [comment]
The cancel order button on the portfolio page works great though!

**Comment by tinybike** [comment]
Ok, it seems like close position only stalls out when I also have an open order in the same market (and same outcome, same direction -- not sure if this matters).  When I cancel my open order, close position works great.

When it stalls out, close position button seems to be forming the trade correctly, I see this in my console:
```
first chunk, clearing order book...
update-trades-in-progress.js?c0cc:136 trading actions: {
  "numShares": "10",
  "limitPrice": "0.5",
  "side": "sell",
  "totalFee": "0.05",
  "totalCost": "4.95",
  "tradeActions": [
    {
      "action": "ASK",
      "shares": "10",
      "gasEth": "0.01393518",
      "feeEth": "0.05",
      "feePercent": "1",
      "costEth": "4.95",
      "avgPrice": "0.495",
      "noFeePrice": "0.5"
    }
  ],
  "tradingFeesEth": "0.05",
  "gasFeesRealEth": "0.01393518",
  "feePercent": "1.020408163265306122"
}
```

**Comment by stephensprinkle** [comment]
Wow, good find.  Taking a look @ why this might be the case now.

**Comment by tinybike** [comment]
Travis build is failing: https://travis-ci.org/AugurProject/augur/builds/222191611

**Comment by stephensprinkle** [comment]
Hey, not quite ready — surfaced one other issue in the shares count on a market page.  Working to resolve that as well.

**Comment by stephensprinkle** [comment]
Closing this for now until the last issues are resolved.

**Pull request closed without merging**